spec/gameobj-data/boxes_and_picking_spec.rb

```diff
@@ -59,12 +88,17 @@
 
     end
 
-    boon_boxes = boon_box_nouns.product(boon_box_prefixes, boon_box_after_names)
+    boon_boxes = boon_box_nouns.product(boon_box_materials, boon_box_descriptions)
 
-    boon_boxes.each do |noun, prefix, after_name|
-      it "recognizes #{prefix} #{noun} #{after_name} as a box" do
-        box = GameObjFactory.item_from_name("#{prefix} #{noun}", noun, after_name)
+    boon_boxes.each do |noun, material, desc|
+      full_boon_box_description = "#{desc} #{material} #{noun}"
+      it "recognizes #{full_boon_box_description} as a box" do
+        box = GameObjFactory.item_from_name(full_boon_box_description)
         expect(box.type).to eq "box"
+      # Removed because long description should not need to be counted - ???
+      #it "recognizes #{desc} #{material} #{noun} as a box" do
+      #  box = GameObjFactory.item_from_name("#{prefix} #{noun}", noun, after_name)
+      #  expect(box.type).to eq "box"
       end
 
       it "recognizes shifting #{prefix} #{noun} #{after_name} as a phased box" do
```

Comment:
You can remove this example block since you moved this into the phased specs below.